docs/monitoring/rsv-gwms-tester.md

```diff
@@ -0,0 +1,223 @@
+
+Ideally, each site supported by your VO(s) should be shown with a green background, which indicates that a Tester job ran at that site recently and successfully. There may be transient failures but if you notice a site in the failed state over multiple days, contact OSG Factory Operations (<osg-gfactory-support@physics.ucsd.edu>) about the failing site, including a link to your Tester RSV results page.
+
+To see detailed information from each probe, click on the probe name in the Metric column.
+
+To see the list of sites that are supported by your VO(s) and are being tested, click the “org.osg.local-gfactory-site-querying-local” link at the bottom of the list of probes. You can also run the probe manually, as described next.
+
+### Listing supported sites manually
+
+To manually run the probe that fetches the list of sites supported by your VO(s), run the following command on your Tester host:
+
+```console
+[root@client ~] # rsv-control --run org.osg.local-gfactory-site-querying-local --host localhost
+```
+
+The probe produces many lines of output, some of which are just about the probe execution itself. But look for lines like this:
+
+```console
+MSG: Updating configuration for host <em>UCSD</em>
+```
+
+The highlighted name is the site name, and there should be one such line per site supported by your VO(s).
+
+Troubleshooting RSV-GWMS-Tester
+-------------------------------
+
+You can find more information on troubleshooting in the [RSV troubleshooting section](https://twiki.grid.iu.edu/bin/view/Documentation/Release3/InstallRSV#Troubleshooting_RSV)
+
+Logs and configuration:
+
+| File Description      | Location               | Comment |
+|:----------------------|:-----------------------|:--------|
+| Condor Cron log files | `/var/log/condor-cron` |         |
+
+| File Description     | Location                                                           | Comment                             |
+|:---------------------|:-------------------------------------------------------------------|:------------------------------------|
+| Metric configuration | `/etc/rsv/metrics/org.osg.local-gfactory-site-querying-local.conf` | To change arguments and environment |
+
+Getting Help
+------------
+
+To get assistance, please use the [this page](../common/help.md).
+
+Reference
+---------
+
+### Certificates
+
+<span class="twiki-macro STARTSECTION">Certificates</span>
+
+| Certificate      | User that owns certificate | Path to certificate                                                           |
+|:-----------------|:---------------------------|:------------------------------------------------------------------------------|
+| Host certificate | `root`                     | `/etc/grid-security/hostcert.pem` &lt;br&gt; `/etc/grid-security/hostkey.pem` |
+
+Find instructions to request a host certificate [here](Documentation/Release3.GetHostServiceCertificates). <span class="twiki-macro ENDSECTION">Certificates</span>
+
+### Networking
```

Comment:
Replace the contents of this section with a link to the `firewall-information` page (link won't actually work until the FirewallInformation doc has been converted, that's OK).